code/network/gtrack.cpp

```diff
@@ -446,6 +463,17 @@ void IdleGameTracker()
 				// send it to the PXO screen				
 				multi_pxo_channel_count_update(channel,num_servers);
 				break;
+
+			case GNT_GAME_PROBE_STATUS:
+				int flags;
+				int next_try;
+
+				memcpy(&flags, inpacket.data, sizeof(int));
+				memcpy(&next_try, inpacket.data+sizeof(int), sizeof(int));
```

Comment:
No need to make these variable sizes explicit?  You would know more than me, I just want to make sure.

Reply:
The PXO code is weird because of it being badly written in the first place requiring extra care needed in porting and compatibility handling. I avoided making some changes so that it would be easier to track against the unmodified retail code. But I also added various safety checks to make sure that various things stay compatible across platforms/compilers. Because of those two things I tend to do stuff like this where I typically wouldn't.

So it looks bad at a glance, but I don't just assume that the data size is always the same and have multiple safety checks to help make sure everything is okay.